---a/net/minecraft/entity/item/EntityMinecartTNT.java
+++b/net/minecraft/entity/item/EntityMinecartTNT.java
@@ -12,69 +12,54 @@
 import net.minecraft.util.EnumParticleTypes;
 import net.minecraft.world.Explosion;
 import net.minecraft.world.World;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class EntityMinecartTNT extends EntityMinecart
-{
+
+public class EntityMinecartTNT extends EntityMinecart {
+
     private int minecartTNTFuse = -1;
     private static final String __OBFID = "CL_00001680";
 
-    public EntityMinecartTNT(World worldIn)
-    {
+    public EntityMinecartTNT(World worldIn) {
         super(worldIn);
     }
 
-    public EntityMinecartTNT(World worldIn, double p_i1728_2_, double p_i1728_4_, double p_i1728_6_)
-    {
+    public EntityMinecartTNT(World worldIn, double p_i1728_2_, double p_i1728_4_, double p_i1728_6_) {
         super(worldIn, p_i1728_2_, p_i1728_4_, p_i1728_6_);
     }
 
-    public EntityMinecart.EnumMinecartType getMinecartType()
-    {
+    public EntityMinecart.EnumMinecartType getMinecartType() {
         return EntityMinecart.EnumMinecartType.TNT;
     }
 
-    public IBlockState getDefaultDisplayTile()
-    {
+    public IBlockState getDefaultDisplayTile() {
         return Blocks.tnt.getDefaultState();
     }
 
-    public void onUpdate()
-    {
+    public void onUpdate() {
         super.onUpdate();
 
-        if (this.minecartTNTFuse > 0)
-        {
+        if (this.minecartTNTFuse > 0) {
             --this.minecartTNTFuse;
             this.worldObj.spawnParticle(EnumParticleTypes.SMOKE_NORMAL, this.posX, this.posY + 0.5D, this.posZ, 0.0D, 0.0D, 0.0D, new int[0]);
-        }
-        else if (this.minecartTNTFuse == 0)
-        {
+        } else if (this.minecartTNTFuse == 0) {
             this.explodeCart(this.motionX * this.motionX + this.motionZ * this.motionZ);
         }
 
-        if (this.isCollidedHorizontally)
-        {
+        if (this.isCollidedHorizontally) {
             double d0 = this.motionX * this.motionX + this.motionZ * this.motionZ;
 
-            if (d0 >= 0.009999999776482582D)
-            {
+            if (d0 >= 0.009999999776482582D) {
                 this.explodeCart(d0);
             }
         }
     }
 
-    public boolean attackEntityFrom(DamageSource source, float amount)
-    {
+    public boolean attackEntityFrom(DamageSource source, float amount) {
         Entity entity = source.getSourceOfDamage();
 
-        if (entity instanceof EntityArrow)
-        {
-            EntityArrow entityarrow = (EntityArrow)entity;
+        if (entity instanceof EntityArrow) {
+            EntityArrow entityarrow = (EntityArrow) entity;
 
-            if (entityarrow.isBurning())
-            {
+            if (entityarrow.isBurning()) {
                 this.explodeCart(entityarrow.motionX * entityarrow.motionX + entityarrow.motionY * entityarrow.motionY + entityarrow.motionZ * entityarrow.motionZ);
             }
         }
@@ -82,120 +82,81 @@
         return super.attackEntityFrom(source, amount);
     }
 
-    public void killMinecart(DamageSource p_94095_1_)
-    {
+    public void killMinecart(DamageSource p_94095_1_) {
         super.killMinecart(p_94095_1_);
         double d0 = this.motionX * this.motionX + this.motionZ * this.motionZ;
 
-        if (!p_94095_1_.isExplosion())
-        {
+        if (!p_94095_1_.isExplosion()) {
             this.entityDropItem(new ItemStack(Blocks.tnt, 1), 0.0F);
         }
 
-        if (p_94095_1_.isFireDamage() || p_94095_1_.isExplosion() || d0 >= 0.009999999776482582D)
-        {
+        if (p_94095_1_.isFireDamage() || p_94095_1_.isExplosion() || d0 >= 0.009999999776482582D) {
             this.explodeCart(d0);
         }
     }
 
-    protected void explodeCart(double p_94103_1_)
-    {
-        if (!this.worldObj.isRemote)
-        {
+    protected void explodeCart(double p_94103_1_) {
+        if (!this.worldObj.isRemote) {
             double d1 = Math.sqrt(p_94103_1_);
 
-            if (d1 > 5.0D)
-            {
+            if (d1 > 5.0D) {
                 d1 = 5.0D;
             }
 
-            this.worldObj.createExplosion(this, this.posX, this.posY, this.posZ, (float)(4.0D + this.rand.nextDouble() * 1.5D * d1), true);
+            this.worldObj.createExplosion(this, this.posX, this.posY, this.posZ, (float) (4.0D + this.rand.nextDouble() * 1.5D * d1), true);
             this.setDead();
         }
     }
 
-    public void fall(float distance, float damageMultiplier)
-    {
-        if (distance >= 3.0F)
-        {
+    public void fall(float distance, float damageMultiplier) {
+        if (distance >= 3.0F) {
             float f2 = distance / 10.0F;
-            this.explodeCart((double)(f2 * f2));
+            this.explodeCart((double) (f2 * f2));
         }
 
         super.fall(distance, damageMultiplier);
     }
 
-    public void onActivatorRailPass(int p_96095_1_, int p_96095_2_, int p_96095_3_, boolean p_96095_4_)
-    {
-        if (p_96095_4_ && this.minecartTNTFuse < 0)
-        {
+    public void onActivatorRailPass(int p_96095_1_, int p_96095_2_, int p_96095_3_, boolean p_96095_4_) {
+        if (p_96095_4_ && this.minecartTNTFuse < 0) {
             this.ignite();
         }
     }
 
-    @SideOnly(Side.CLIENT)
-    public void handleHealthUpdate(byte p_70103_1_)
-    {
-        if (p_70103_1_ == 10)
-        {
-            this.ignite();
-        }
-        else
-        {
-            super.handleHealthUpdate(p_70103_1_);
-        }
-    }
-
-    public void ignite()
-    {
+    public void ignite() {
         this.minecartTNTFuse = 80;
 
-        if (!this.worldObj.isRemote)
-        {
-            this.worldObj.setEntityState(this, (byte)10);
+        if (!this.worldObj.isRemote) {
+            this.worldObj.setEntityState(this, (byte) 10);
 
-            if (!this.isSilent())
-            {
+            if (!this.isSilent()) {
                 this.worldObj.playSoundAtEntity(this, "game.tnt.primed", 1.0F, 1.0F);
             }
         }
     }
 
-    @SideOnly(Side.CLIENT)
-    public int getFuseTicks()
-    {
-        return this.minecartTNTFuse;
-    }
-
-    public boolean isIgnited()
-    {
+    public boolean isIgnited() {
         return this.minecartTNTFuse > -1;
     }
 
-    public float getExplosionResistance(Explosion p_180428_1_, World worldIn, BlockPos p_180428_3_, IBlockState p_180428_4_)
-    {
+    public float getExplosionResistance(Explosion p_180428_1_, World worldIn, BlockPos p_180428_3_, IBlockState p_180428_4_) {
         return this.isIgnited() && (BlockRailBase.isRailBlock(p_180428_4_) || BlockRailBase.isRailBlock(worldIn, p_180428_3_.up())) ? 0.0F : super.getExplosionResistance(p_180428_1_, worldIn, p_180428_3_, p_180428_4_);
     }
 
-    public boolean func_174816_a(Explosion p_174816_1_, World worldIn, BlockPos p_174816_3_, IBlockState p_174816_4_, float p_174816_5_)
-    {
+    public boolean func_174816_a(Explosion p_174816_1_, World worldIn, BlockPos p_174816_3_, IBlockState p_174816_4_, float p_174816_5_) {
         return this.isIgnited() && (BlockRailBase.isRailBlock(p_174816_4_) || BlockRailBase.isRailBlock(worldIn, p_174816_3_.up())) ? false : super.func_174816_a(p_174816_1_, worldIn, p_174816_3_, p_174816_4_, p_174816_5_);
     }
 
-    protected void readEntityFromNBT(NBTTagCompound tagCompund)
-    {
+    protected void readEntityFromNBT(NBTTagCompound tagCompund) {
         super.readEntityFromNBT(tagCompund);
 
-        if (tagCompund.hasKey("TNTFuse", 99))
-        {
+        if (tagCompund.hasKey("TNTFuse", 99)) {
             this.minecartTNTFuse = tagCompund.getInteger("TNTFuse");
         }
     }
 
-    protected void writeEntityToNBT(NBTTagCompound tagCompound)
-    {
+    protected void writeEntityToNBT(NBTTagCompound tagCompound) {
         super.writeEntityToNBT(tagCompound);
         tagCompound.setInteger("TNTFuse", this.minecartTNTFuse);
     }
-}
+}
